### ISSUE TITLE
softmaxproto loss for binary classification

### ISSUE THREAD

**Issue by ukemamaster** [clovaai/voxceleb_trainer#189]
Hi @joonson, 
I am trying to adapt the code to train a binary classifier. I am getting a validation accuracy of up to 99%, but the loss values are very very high (in the order of 10^9). Is it normal? Is it a good idea to use **softmaxproto** loss for binary classification problem? if not, why?
